Address improvements to "Configure Session Time Out and Remember Me" doc (#2529)
wso2/docs-is: en/docs/guides/login/session-timeout.md

This is the duration in weeks for which WSO2 Identity Server should remember an SSO session given that you have selected the Remember Me option in the WSO2 Identity Server login screen. The default value is **2 weeks**.

---

## Configuration

> To configure the **Session Timeout** and **Remember Me period** in tenant wise, follow the steps below:

Reviewer: ```suggestion
To configure the **Session Timeout** and **Remember Me period**  tenant wise, follow the steps below:
```

> 1.  Start the Identity Server and login to the management console.

Reviewer: ```suggestion
1.  Start the Identity Server and log in to the management console (`https://<IS_HOST>:<PORT>/carbon`).
```

2.  Click **Resident** under **Identity Providers** on the **Main** tab.

> 3.  Fill in the fields as seen below to configure the **Idle Session Time Out**

Reviewer: ```suggestion
3.  Fill in the fields as seen below to configure **Idle Session Time Out**
```

    and **Remember Me Period**.  
    
    ![session-time-out](../../assets/img/guides/session-time-out-config.png) 
    
    !!! Tip "Configure timeout values globally" 
        The global configuration can be added to the

>         `<IS_HOME>/repository/conf/deployment.toml` file as shown below. The `idle_session_timeout` property that defines the session time out period. The `remember_me_session_timeout` property defines the period of time for which the WSO2 Identity Server will remember the user's SSO session.

Reviewer: ```suggestion
        `<IS_HOME>/repository/conf/deployment.toml` file as shown below. The `idle_session_timeout` property defines the session time out period. The `remember_me_session_timeout` property defines the period of time for which the WSO2 Identity Server will remember the user's SSO session.
```
